packages/web-components/src/components/rux-slider/rux-slider.tsx

```diff
@@ -193,6 +198,10 @@ export class RuxSlider implements FormFieldInterface {
         this.ruxInput.emit()
     }
 
+    private _onChange() {
+        this.ruxChange.emit()
+    }
+
     private _onBlur = () => {
```

Comment:
@markacianfrani do you know why this is an arrow function? I don't think it's a problem I'm just curious - hell it might have been me that added it 🤷